hrms/hr/doctype/expense_claim/bulk_payment.py

```diff
@@ -0,0 +1,190 @@
+            if paid_amount == 0:
+                break
+
+        if actual_paid_amount != float(row["Amount"]):
+            frappe.throw(_(f"Paid Amount not matching with the expense claim amount"))
+
+        party_details = get_party_details(
+            employee_doc.company, "Employee", employee_doc.name, frappe.utils.nowdate()
+        )
+
+        company_bank_account_exists = frappe.db.get_list(
+            "Bank Account",
+            filters={
+                "currency": "INR",
+                "is_company_account": 1,
+                "company": CompanyNames.SUITE42,
+                "Bank": CompanyConstants.KOTAK_BANK,
+                "account": "Kotak - 5946999821 - Suite42",
+            },
+            pluck="name",
+        )
+
+        if not company_bank_account_exists:
+            frappe.throw(_(f"For Company {CompanyNames.SUITE42} No INR Bank Account present"))
+        else:
+            # Currently Allowing only One Account
+            company_bank_account_doc = frappe.get_doc(
+                "Bank Account", company_bank_account_exists[0]
+            )
+
+        bank_cash_account = frappe.get_doc("Account", company_bank_account_doc.account)
+
+        bank_account_currency = company_bank_account_doc.currency
+
+        paid_to = Suite42ApplicationConfig.get_json_value_without_error("PAYABLE_ACCOUNTS")[
+            CompanyNames.SUITE42
+        ]["reimbursement_payable_account"]
+
+        paid_to_account_currency = get_account_details(paid_to, frappe.utils.nowdate()).get(
+            "account_currency"
+        )
+        refrence_date = datetime.strptime(row["Maker DateTime"].split(" ")[0], "%d/%m/%Y").date()
+        payment_entry_doc = frappe.get_doc(
+            {
+                "doctype": "Payment Entry",
+                "docstatus": 1,
+                "payment_type": "Pay",
+                "mode_of_payment": row["Product Code"],
+                "party_type": "Employee",
+                "party": employee_doc.name,
+                "party_name": employee_doc.employee_name,
+                "paid_from": bank_cash_account,
+                "paid_from_account_currency": bank_account_currency,
+                "paid_to": paid_to,
+                "paid_to_account_currency": paid_to_account_currency,
+                "paid_amount": actual_paid_amount,
+                "total_allocated_amount": actual_paid_amount,
+                "received_amount": actual_paid_amount,
+                "unallocated_amount": 0,
+                "references": refrence_list,
+                "reference_no": row["UTR SrNo"],
+                "reference_date": refrence_date,
+            }
+        )
+        payment_entry_doc.insert(ignore_permissions=True).name
+        for expense_claim in expense_claim_list:
+            expense_claim_doc = frappe.get_doc("Expense Claim", expense_claim)
+            expense_claim_doc.is_paid = 1
+            expense_claim_doc.status = ExpenseClaimConstants.PAID
```

Comment:
expense_claim_doc account should be equalt to paid_to_account

Reply:
Not Done here, Checked at the start while creating payment entry